libs/platform/src/lib/value-help-dialog/value-help-dialog/value-help-dialog.component.html

```diff
@@ -100,7 +100,6 @@ <h5 fd-title class="selected-and-condition-title" [id]="id + '-selected-item'">
                 <div class="fdp-value-help-dialog__actions-row">
                     <button
                         fd-button
-                        [compact]="contentDensity | isCompactDensity"
```

Comment:
Does it means we cannot specify `contentDensity` for the vhd component and always have cozy?